### PR TITLE
[terra-signature] Example update

### PR DESCRIPTION
**Pull request by MadanKumarGovindaswamy**
### Summary

**What was changed:**

Updated terra signature example to include text and image signature

**Why it was changed:**

terra signature example does not support text and image. updated current example for the illustration

### Testing

This change was tested using:

- [ ] WDIO
- [ ] Jest
- [ ] Visual testing (please attach a screenshot or recording)
- [ ] Other (please describe below)
- [x] No tests are needed

### Reviews

In addition to engineering reviews, this PR needs:

- [ ] UX review
- [ ] Accessibility review
- [x] Functional review

### Additional Details

**This PR resolves:**

UXPLATFORM-XXXX 

---

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

**Comment by rbsree** [comment]
Hi @MadanKumarGovindaswamy, we need to remove tabindex="0" attribute from the <canvas> tag.

**Review by allanxavierm — APPROVED** [review]

**Comment by rbsree** [comment]
Hi @MadanKumarGovindaswamy, as discussed mentioning the issues found for Terra Signature.
Issues:
1. Pressed tab, expecting to navigate to the signature input box, but VO announces it as - Type tab panel, you are currently on the tab panel. This is because tabindex="0" is added to the "tabpanel" element. **(VO, JAWS)**
2. Pressed tab again to enter the signature input box, VO read it as edit text blank, type, tab panel. You are currently on text area.  **(VO, JAWS)** - Provide a contextual label to the <textarea> field.
3. With the typed name, when the user triggers the "Add" or "Clear" button then the signature is getting added or cleared but the same is not exposed to screen reader users. Add live region message to update the same to screen reader users.

Keyboard:
1. The Clear button is partially obscured and does not fully display the focus indicator **- Safari**
2. When using a resolution of 320x256 or 400% zoom, if the user's name is lengthy then it may not be fully visible.

**Comment by MadanKumarGovindaswamy** [comment]
Hi @rbsree ,
Have addressed the discussed issues in signature.
Thanks

File: packages/terra-core-docs/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Updated
+  * Updated terra signature example to include text and image signature.
+
 * Changed
   * Fixed typos and imports in `terra-list` docs.
```
Comment (sycombs) [inline packages/terra-core-docs/CHANGELOG.md:9]: nit: for consistency, can we change this heading to "Changed" and combine it with the changelog entry on line 8? also can we add backticks around terra-signature?

```suggestion
* Changed
  * Fixed typos and imports in `terra-list` docs.
  * Updated `terra-signature` example to include text and image signature.
```

**Review by sycombs — APPROVED** [review]

**Comment by rbsree** [comment]
+1 for the Accessibility review.
The above recommended suggestion is implemented expect the following issue:
- Pressed tab, expecting to navigate to the signature input box, but VO announces it as - Type tab panel, you are currently on the tab panel. This is because tabindex="0" is added to the "tabpanel" element. **(VO, JAWS)**

**Pull request merged**